### DISCUSSION TITLE
Live stream can't connect, always have to update

### DISCUSSION THREAD

**Discussion by robbedoeske** [AzuraCast/AzuraCast#6242] [Q&A]
Livestream works perfect but after one day I can't connect anymore. I have to update AzuraCast and then I can login??

**Comment by omarelifaz** [comment]
I have the same problem, suddenly it stopped working, I can't connect to the live broadcast, when the autodj is working. The player shows me that the station is turned off, but it only plays audio from the autodj. Could someone please help me with the solution... Thanks...
AzuraCast • v0.17.7 Stable • Docker • PHP 8.2
